### ISSUE TITLE
www/index.html should not be required for imported projects

### ISSUE THREAD

**Issue by GoogleCodeExporter** [erhanselman/mobile-web-development-with-phonegap#50]
```
From 
https://groups.google.com/forum/#!topic/eclipse-plugin-for-phonegap/csxVVSlaPMU

I installed mds and was importing my phonegap project but failed with
error messages says "must include an index.html file".
In fact, I moved my index.html to /assets/www/common and changed the
code of loading index file like below:
super.loadUrl(String.format("file:///android_asset/www/common/
index.html"));
I think there is no need to validate whether index.html is there
because developer should feel free to organize his sources.
```

Original issue reported on code.google.com by `paul.beu...@gmail.com` on 27 Nov 2011 at 4:15

**Comment by GoogleCodeExporter** [comment]
```
The 1.2.6 wizard will include a Pure Import check box to create the www 
directory exactly from the specified directory without any changes.
```

Original comment by `paul.beu...@gmail.com` on 16 Dec 2011 at 5:57
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Fixes available with 1.2.6
```

Original comment by `paul.beu...@gmail.com` on 21 Dec 2011 at 3:57
- Changed state: **Fixed**
